dashboard/templates/base.html

```diff
@@ -23,7 +23,7 @@
     {% include "su/is_su.html" %}
     <header class="row border-between">
         <div class="col-md-4 text-left">
-            <a href="/courses/{{ current_user_course_id }}/">{{settings.PROJECT_NAME}}</a> {% if course_id %} : {{ course_name }} {% endif %}
+            <a href="/courses/{{ current_user_course_id }}/">My Learning Analytics</a> {% if course_id %} : {{ course_name }} {% endif %}
         </div>
```

Comment:
What was the need to remove it? I feel this is good from before without hardcoding it.

Reply:
I just reverted the entire commit. I created a new issue to re-add this fix that was un-related to this.